app.js

```diff
@@ -1,10 +1,25 @@
 const express = require("express");
 const app = express();
-const { getTopics } = require("./db/controllers/news.controller");
+const {
+  getTopics,
+  getArticleById,
+} = require("./db/controllers/news.controller");
 
 app.use(express.json());
 
 app.get("/api/topics", getTopics);
+app.get("/api/articles/:article_id", getArticleById);
+
+app.use((err, req, res, next) => {
+  console.log(err.code);
```

Comment:
remove console.log()

Reply:
Thank you! Should I ask for pr again after removing of consoles?